javascript/uber_search.js

```diff
@@ -52,28 +57,56 @@ var UberSearch = function(data, options){
     view: {
       renderResults: renderResults,
       buildResult: options.buildResult || buildResult,
-      keypressInput: options.search ? searchField.input : outputContainer.view
+      keypressInput: options.search ? searchField.input : null
     }
   })
 
 
   // BEHAVIOUR
 
+  // Show the pane when the select element is clicked
+  $(outputContainer.view).on('click', function(event){
+    if (outputContainer.view.hasClass('disabled')) { return }
+
+    pane.show()
+  })
+
+  // Show the pane if the user was tabbed onto the trigger and pressed enter, space, or down arrow
+  $(outputContainer.view).on('keyup', function(event){
+    if (outputContainer.view.hasClass('disabled')) { return }
+
+    if (event.which === 32 || event.which === 40 && pane.isClosed()){
+      pane.show()
+      return false
+    }
+    else if (event.which === 13){ // toggle pane when enter is pressed
+      pane.toggle()
+      return false
+    }
+  })
+
   // When the pane is opened
   $(pane).on('shown', function(){
     search.clear()
-    markSelected()    
+    markSelected() 
     view.addClass('open')
    
     if (options.search) {
       $(searchField.input).focus()
     } else {
-      pane.view.find("ul.results li:first").focus()
+      pane.view.find("ul.results li:first > a").focus()
```

Comment:
Be careful here. Results rendering can be customized so it is not guaranteed they will always have an `a` tag. Why can't we focus the `<li>` still?

Reply:
@njakobsen It doesn't seem `li` elements are focusable.  Here are a couple of references that helped me there:

https://stackoverflow.com/a/1600194
https://gist.github.com/jamiewilson/c3043f8c818b6b0ccffd

Reply:
Here's a codepen I made showing it's definitely focussable https://jsfiddle.net/germdn78/.

Reply:
I was running into some issues.  With it in uber_select.  I'll look into it.